src/View/Form/ArrayContext.php

```diff
@@ -206,6 +206,27 @@ public function isRequired($field)
         return (bool)$required;
     }
 
+    /**
+     * Get field length from validation
+     *
+     * In this context class, this is simply defined by the 'maxlength' array.
+     *
+     * @param string $field A dot separated path to check required-ness for.
+     * @return int|false
```

Comment:
for new API methods i would rather want to see nullable 

    int|null

also for future php typehinting